superset/views/database/views.py

```diff
@@ -92,34 +98,6 @@ class DatabaseView(DatabaseMixin, SupersetModelView, DeleteMixin, YamlExportMixi
 
     yaml_dict_key = "databases"
 
-    @expose("/show/<pk>", methods=["GET"])
-    @has_access
-    @deprecated(eol_version="5.0.0")
-    def show(self, pk: int) -> FlaskResponse:
-        """Show database"""
-        return super().show(pk)
-
-    @expose("/add", methods=["GET", "POST"])
```

Comment:
Is this (as well as `/edit/<pk>` and `/delete/<pk>`) deprecated or actually removed? I couldn't find their alternative form (unlike `/show/<pk>`) in your PR. The reason I'm asking is removing a legacy endpoint would be considered a breaking change.

Reply:
those are all in on `DeprecateModelViewMixin`, removing them in favour of the mixin